Formula/aerc.rb

```diff
@@ -0,0 +1,34 @@
+class Aerc < Formula
+  include Language::Python::Virtualenv
+
+  desc "The world's best email client"
+  homepage "https://aerc-mail.org/"
+  url "https://git.sr.ht/~sircmpwn/aerc/archive/0.1.1.tar.gz"
+  sha256 "eb9f6cf6f77b1eef25d04d7c7cd05fdaebd3d41cd47fa95230b222acac94fc66"
+
+  depends_on "go" => :build
+  depends_on "scdoc" => :build
+
+  depends_on "dante"
+  depends_on "python"
+  depends_on "w3m"
+
+  resource "colorama" do
+    url "https://files.pythonhosted.org/packages/76/53/e785891dce0e2f2b9f4b4ff5bc6062a53332ed28833c7afede841f46a5db/colorama-0.4.1.tar.gz"
+    sha256 "05eed71e2e327246ad6b38c540c4a3117230b19679b875190486ddd2d721422d"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install resources
+    inreplace "contrib/plaintext.py", "/usr/bin/env python3", "#{libexec}/bin/python3"
```

Comment:
Why is this necessary? `python3` is in Homebrew PATH, so it should work as it is.

Reply:
My understanding is that it's poor form to install a python dependency (colorama) into the Homebrew global Python environment and so instead I put it in a virtualenv.  These scripts have pip dependencies and by default are directly exec'd by the main golang executable.  Without manipulating the shebang, they run in the global homebrew env which lacks colorama.

Alternatively, I could wrap execution of the main executable with a shell script that first activates the virtualenv before launching the golang bin.  If there are other preferred method(s) please let me know.

Reply:
@notpeter Would it be possible to just install the python resources into `libexec` without a virtualenv and instead set the `PYTHONPATH` accordingly so it can be found? This feels like the responsibility of the upstream software project's packaging so it'd be good to create a feature request/patch for something similar to be added in future.

Reply:
@MikeMcQuaid Yeah, official upstream stance is basically "install requirements into your global python or activate a appropriate venv first."  It's definitely a little hacky that their go application just execs some bundled python scripts and displays the output inline, but it's not insane, feels actually kinda UNIXy.

Manipulating the PYTHONPATH would require wrapping every execution of the main go bin in a script, no?  That feels similarly.

Reply:
> Manipulating the PYTHONPATH would require wrapping every execution of the main go bin in a script, no? That feels similarly.

That's pretty easy for us to do. Grep for `bin.env_script_all_files`.